frontend/src/scenes/insights/ActionFilter/ActionFilterRow.js

```diff
@@ -184,6 +182,9 @@ export function ActionFilterRow({
             {visible && (
                 <div className="ml">
                     <PropertyFilters
+                        buttonStyle={{
+                            maxWidth: 'calc(100% - 24px)',
```

Comment:
Nit: It's quite helpful for future reference to detail why 24px, can make it easier for someone else with no context too.